aiohttp/web_protocol.py

```diff
@@ -137,7 +137,7 @@ class RequestHandler(BaseProtocol):
                  '_waiter', '_error_handler', '_task_handler',
                  '_upgrade', '_payload_parser', '_request_parser',
                  '_reading_paused', 'logger', 'access_log',
-                 'access_logger', '_close', '_force_close')
+                 'access_logger', '_close', '_force_close', 'handle_request')
```

Comment:
This change looks irrelevant.

Reply:
There are many tests mock the `handle_request`(not `_handle_request`), but I could not find where call the `handle_request` in aiohttp. Only some docstring mentions it.

https://github.com/aio-libs/aiohttp/blob/dc39183d4351c2fc87dbfa9bcf938567d754571b/aiohttp/web_protocol.py#L97-L99

Reply:
Heh, it is an excellent demonstration of mocks weakness.

`handle_request` is `self._manager.request_handler` I guess. 
But our mocked tests didn't fail despite the API was changed.

The fix should be done in a separate PR, this not existing yet pull request is a blocker for your work.

Reply:
Due to the `asyncio` changes, we could not assign a value to `handle_request`. Actually this block my progress. I need to fix all like https://github.com/aio-libs/aiohttp/blob/dc39183d4351c2fc87dbfa9bcf938567d754571b/tests/test_web_protocol.py#L403

change above to

```Python
with mock.patch.object(web.RequestHandler, 'handle_request', create=True, new=handle):
    pass
```